Ginger/GingerCore/Actions/ActSikuli.cs

```diff
@@ -584,5 +602,18 @@ private List<int> GetCustomResolutionValues()
 
             return lstVal;
         }
+
+        private void SetProcessAsPerVE()
+        {
+            int countSimilar = ActiveProcessWindows.Where(p => p.Contains(ProcessNameVEForSikuliOperation)).Count();
+            if (countSimilar == 1)
```

Comment:
If more than 1 processes are open of the title given in VE, it will not set it. 
Please use .Any() in the if with where query which is used on line 608